charts/consul/values.yaml

```diff
@@ -1131,6 +1131,41 @@ server:
     # @type: string
     caCert: null
 
+  # Settings for potentially limiting timeouts, rate limiting on clients as well
```

Comment:
Nit. Could you add a link to the rate limits configuration?

https://developer.hashicorp.com/consul/docs/agent/config/config-files#request_limits

Reply:
Great call.  Added here: https://github.com/hashicorp/consul-k8s/pull/2170/commits/7ab9a14a946e9a6916f5aed01755555c55241101#diff-9d190488582b6dfdeb91fd565c91a267e5dc1c1b925a174174aade2a503f5373R1141